clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentials.groovy

```diff
@@ -184,6 +186,11 @@ class GoogleNamedAccountCredentials implements AccountCredentials<GoogleCredenti
       return this
     }
 
+    Builder regionsToManage(List<String> regionsToManage, List<String> defaultRegions) {
+      this.regionsToManage = regionsToManage ?: defaultRegions
```

Comment:
If `regionsToManage` is an empty list (i.e., don't index anything for this account), won't this default to `defaultRegions`? I think you need to explicitly check that `regionsToManage` is `null`.